app/presenters/register_api/single_application_presenter.rb

```diff
@@ -0,0 +1,245 @@
+          candidate: {
+            id: "C#{application_form.candidate.id}",
+            first_name: application_form.first_name,
+            last_name: application_form.last_name,
+            date_of_birth: application_form.date_of_birth,
+            nationality: application_choice.nationalities,
+            domicile: application_form.domicile,
+            uk_residency_status: uk_residency_status,
+            uk_residency_status_code: uk_residency_status_code,
+            fee_payer: provisional_fee_payer_status,
+            english_main_language: application_form.english_main_language,
+            english_language_qualifications: application_form.english_language_qualification_details,
+            other_languages: application_form.other_language_details,
+            disability_disclosure: application_form.disability_disclosure, # do we need this?
+            gender: application_form.equality_and_diversity.try(:[], 'sex'),
+            disabilities: application_form.equality_and_diversity.try(:[], 'disabilities'),
+            ethnic_group: application_form.equality_and_diversity.try(:[], 'ethnic_group'),
+            ethnic_background: application_form.equality_and_diversity.try(:[], 'ethnic_background'),
+          },
+          contact_details: contact_details,
+          course: course_info_for(application_choice.offered_option),
+          provider: provider_info_for(application_choice.offered_option.course.provider),
+          qualifications: qualifications,
+          hesa_itt_data: hesa_itt_data,
+        },
+      }
+    end
+
+  private
+
+    attr_reader :application_choice, :application_form
+
+    def status
+      application_choice.status
+    end
+
+    def uk_residency_status
+      return 'UK Citizen' if application_choice.nationalities.include?('GB')
+
+      return 'Irish Citizen' if application_choice.nationalities.include?('IE')
+
+      return application_form.right_to_work_or_study_details if application_form.right_to_work_or_study_yes?
+
+      'Candidate needs to apply for permission to work and study in the UK'
+    end
+
+    def uk_residency_status_code
+      return 'A' if application_choice.nationalities.include?('GB')
+      return 'B' if application_choice.nationalities.include?('IE')
+      return 'D' if application_form.right_to_work_or_study_yes?
+
+      'C'
+    end
+
+    def provisional_fee_payer_status
+      return UCAS_FEE_PAYER_CODES['SLC,SAAS,NIBd,EU,Chl,IoM'] if provisionally_eligible_for_gov_funding?
+
+      UCAS_FEE_PAYER_CODES['Not Known']
+    end
+
+    def provisionally_eligible_for_gov_funding?
+      return true if (PROVISIONALLY_ELIGIBLE_FOR_GOV_FUNDING_COUNTRY_CODES & application_choice.nationalities).any?
+
+      (EU_EEA_SWISS_COUNTRY_CODES & application_choice.nationalities).any? &&
+        application_form.right_to_work_or_study_yes? &&
+        application_form.uk?
+    end
+
+    def course_info_for(course_option)
```

Comment:
Sorry to come late to this! Does anyone else feel a bit funny about Apply passing on data from TTAPI? This will now be a third copy of the data. Do we risk course data going stale in register — what if the provider made a mistake in the subject codes and updated it in Publish? Or the course name changed? Or if these details changed in Apply, would this application be marked as changed in the Register API? (That might be fiddly). Maybe a snapshot at the point of application is desirable, though? cc @gpeng @AbigailMcP